queries/vue/injections.scm

```diff
@@ -49,6 +49,30 @@
   (#eq? @_ts "ts")
 )
 
+; <script lang="jsx">
+(
+  (script_element
+    (start_tag
+      (attribute
+        (attribute_name) @_lang
+        (quoted_attribute_value (attribute_value) @_js)))
+    (raw_text) @javascript)
+  (#eq? @_lang "lang")
+  (#eq? @_js "jsx")
+)
+
+; <script lang="tsx">
+(
+  (script_element
+    (start_tag
+      (attribute
+        (attribute_name) @_lang
+        (quoted_attribute_value (attribute_value) @_ts)))
```

Comment:
can we please unify those three queries?

Reply:
Yes

Reply:
I think it's ok to invent a new directive for this if necessary. This could be used later also for other web languages. This might use a locally specified mapping or a let nvim-treesitter resolve automatically commonly known file endings

Reply:
Oh very nice